### DISCUSSION TITLE
Cloudstrapper instance

### DISCUSSION THREAD

**Discussion by mickymkumar** [magma/magma#6207] [Q&A]
How do you create Cloudstrapper instance on Windows?

**Comment by karthiksubraveti** [comment]
Cc @arunuke

**Comment by arunuke** [comment]
The current Cloudstrapper available from the experimental folder (https://github.com/magma/magma/tree/master/experimental/cloudstrapper) would work with Ubuntu and possibly other Debian variants, but might need tweaks to work on other Linux distros and Windows (unteted). The fundamental idea is still the same, to configure a running instance with the dependencies and then snapshot it.
